Environment.Build.props

```diff
@@ -29,7 +29,7 @@
   </PropertyGroup>
   
   <PropertyGroup>
-    <LangVersion Condition="'$(LangVersion)' == ''">7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
can't this be used for default in all project ? so we could only have single reference to the LangVersion HERE, and nowhere else

Reply:
I wish but I ran into issues trying to make that work with non sdk style projects. Maybe I was holding it wrong? 

https://github.com/xamarin/Xamarin.Forms/blob/d5d1bbf0ca61601c59187cc2037b437bead9913a/Stubs/Xamarin.Forms.Platform.iOS/Xamarin.Forms.Platform.iOS%20(Forwarders).csproj#L15